src/tests/ftest/dfuse/read.yaml

```diff
@@ -0,0 +1,22 @@
+hosts:
+  test_servers: 1
```

Comment:
Better to explicitly set clients. One day we need to only use localhost as the runner
```suggestion
hosts:
  test_servers: 1
  test_clients: 1
```